core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
i'm not saying to comment that line, that line is ok.
if you start another wh_cache with `[{origin_bindings, []}]` then you will get a *.*.database.*. this is what i'm trying to prevent.
you also need to match [] and not only [[]].